documentation/content/Components/photoswipe-instagramp-gallery.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: Gallery -- Use Instagram
+authors: Talha Mansoor, Pablo Iranzo Gómez
+Tags: nuances, images, gallery, instagram
+Date: 2020-02-06
+Slug: photoswipe-gallery-using-instagram
+Category: Components
+---
+
+Similar to the use case defined in [Photogallery]({filename}photogallery.md), Pelican has code in place for showing instragram galleries.
+
+## Article contents
+
+To enable the plugin, just define a similar div inside your article for each gallery:
+
+```yaml
+---
+author: Pablo Iranzo Gómez
+title: Instagram
+tags: python, redken, descuenbot, deal, chollo, bargain, rabbate, soldes
+date: 2020-01-27 00:35:00 +0100
+comments: true
+category: blog
+description:
+lang: en
+---
+Single image
+<div class="elegant-instagram" data-instagram-id="B7yh4IdItNd"></div>
+Multiple image
+<div class="elegant-instagram" data-instagram-id="BwWo35fAcR3"></div>
+```
+
+For reference, the `data-instagram-id` is taken from Instagram picture URL, for example:
+
+`https://www.instagram.com/p/OzF8OwS43q/` will mean adding to the div the parameter `data-instagram-id=OzF8OwS43q`.
+
+!!! hint "`data-instagram-id` defines either picture or set of pictures"
+
+    Note that `data-instagram-id` is the part in the url of a post, that can be a single or multiple pictures 'post' and in both case the library will show them.
+
+The above `article` code will then render as this:
+
+Single image
+
+<div class="elegant-instagram" data-instagram-id="B7yh4IdItNd"></div>
+Multiple image
```

Comment:
Place multiple image example above the single image.

Reply:
done